package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "@types/jest": "^26.0.23"
```

Comment:
Why there is a single dependency is here? I think something is wrong with this file.

Reply:
haha, this is the package.json in the root of the ml/frm project - not the one inside the Payment Platform Adapter. I was very confused for a second.